index.html

```diff
@@ -52,7 +52,113 @@ <h1>  I’m Lalla <br>
               </ul>
           
           </section>
-         
+          <div class="grid-container">
+            <section class = "works">
+                <div class="job">
```

Comment:
- [ ] There are still of details that you missed to implement to match the Figma Design 🔽 please work on them.

## Your Design
![2023-03-25_215654](https://user-images.githubusercontent.com/104319462/227744497-b055d63f-dc82-46ed-a059-d5a5c177816b.png)

## Figma Design
![2023-03-25_220151](https://user-images.githubusercontent.com/104319462/227744488-3fcbd3a7-58ef-48cd-8fa1-e266d906ffe9.png)